containter_host.tf

```diff
@@ -78,11 +83,27 @@ resource "null_resource" "cluster" {
     destination = "/home/${var.user}/zookeeper.keystore.jks"
   }
 
+  provisioner "file" {
```

Comment:
what if pem files are not used and thus not avaialable. So this should be made optional

Reply:
but the same goes for the trust-store and other certificates. We've accepted that this only works with mTLS enabled, so this shouldn't be a problem then? If it is, then we should change it for the others as well.

Reply:
TLS should be optional or driven by the availability of pem. Does this PR always assume TLS?

Reply:
Hi Andy, this whole repo does assume that mTLS is mandatory. E.g. look to: docker run command variable: --env KAFKA_ZOOKEEPER_PROTOCOL="SSL" \
That is hardcoded SSL, right?

Reply:
Johan, indeed. @krishna-prasad-s any comments on this, since you brought it up? How much work to make SSL optional?

Reply:
so, for configurability
1. SSL (on - off)
2. certificates to be either pem or jks.
I think in the short term we should be able to do 2. where instead of hardcoding for .jks or .pem this should be derived from a var. and then the type should be pem.  @rik-v could you do that?

Reply:
we can tackle the SSL (on-off) seperately